pages/how-to-contribute.md

```diff
@@ -0,0 +1,149 @@
+---
+title: How To Contribute
+permalink: /how-to-contribute/
+redirect_from: /contributor/
+---
+
+Contributions are welcome to all Apache Accumulo repositories. While most contributions are code,
+there are other ways to contribute to Accumulo:
+
+* answer questions on mailing lists
+* review pull requests
+* verify and test new releases
+* update the Accumulo website and documentation
+
+Contributions are reviewed (via GitHub pull requests) by
+the community before being merged by a committer.
+
+This document provides basic instructions for contributing to Accumulo.  If you are looking for more information, check out the more comprehensive [contributor guide](/contributors-guide/).
+
+## Issues
+
+Any contribution should have a corresponding issue. Accumulo uses [JIRA] for issue tracking. Before creating an issue,
+you will need to create an [Apache JIRA account][jira-signup]. If you need help finding an issue to work on, check out
+the [open issues labeled for newbies][newbie-issues] or [contact us][contact].
+
+## Repositories
+
+Contributions can be made to the following repositories. While the general contribution workflow is
+described below, repositories have special instructions in their `CONTRIBUTING.md` file which can be
+viewed by clicking on the `C` in the Links column below.
```

Comment:
I am not a fan of single letter links. Could put the name of the file.

Reply:
I am following [what was done in Fluo](https://fluo.apache.org/how-to-contribute/) but if you don't think it's clear I am sure others will.  I am OK if `CONTRIBUTING.md` is used instead.

Reply:
Yeah its just annoying to click a single letter.  Even the word "link" would be better.  Or "contribute" is shorter.

Reply:
I'll update the links to say 'contribute' without. But when testing the links just now I noticed that the accumulo-website link is broken. Going over to the accumulo-website github page I see that there is not a CONTRIBUTING.md file.  Has there been one in the past? And if now, what should this link connect to?

Reply:
Never mind, I see that there will be a new CONTRIBUTING.md when the PR gets pushed.

Reply:
Pushed the change up to Github